HandyHansel/Commands/KarmaCommands.cs

```diff
@@ -50,7 +65,9 @@ public async Task GiveKarma(CommandContext context, DiscordMember member)
             }
 
             provider.AddKarma(member.Id, context.Guild.Id, karmaToAdd);
-            await context.Message.CreateReactionAsync(DiscordEmoji.FromName(context.Client, ":arrow_up:"));
+            DiscordEmbed response = new DiscordEmbedBuilder()
+                .WithDescription($"{member.Mention}, you have been bestowed {karmaToAdd} karma.");
+            await context.RespondAsync(embed: response);
```

Comment:
Add better response while deleting the user's message to preserve some level of anonymity.